app/modules/media/media_controller.py

```
@@ -0,0 +1,271 @@
+            if attachment.message_id:
+                access_granted = await self._check_attachment_access(
+                    attachment.message_id
+                )
+                if not access_granted:
+                    raise HTTPException(status_code=403, detail="Access denied")
+
+            # Get the file data from storage
+            file_data = await self.media_service.get_attachment_data(attachment_id)
+
+            # Create streaming response
+            file_stream = io.BytesIO(file_data)
```

Ruff (0.11.9)

Local variable `file_stream` is assigned to but never used

Remove assignment to unused variable `file_stream`

(F841)